webapp/app.py

```diff
@@ -38,4 +66,10 @@ def convert_to_kebab(kebab_input):
     words = re.findall(
         r"[A-Z]?[a-z]+|[A-Z]{2,}(?=[A-Z][a-z]|\d|\W|$)|\d+", kebab_input
     )
+
     return "-".join(map(str.lower, words))
+
+
+def remove_special_chars(text):
```

Comment:
It looks like you added this function twice - once here and once in `get_job_feed.py`. If you need it here you should import it from there.